README.md

```diff
@@ -1 +1,46 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+This project is about building a Portfolio using the mobile-first approach. This is the first step of the whole project for my portfolio. According to project requirements, we must follow one of the designs from provided templates to design the website and I have decided to follow the "Template 2" design and look from Figma.
+
+
+## Build with
+- HTML
+- CSS
+
+## Live Demo
+https://github.com/developertariq/Portfolio
+
+
+## Getting Started
+To get a local copy up and download from https://github.com/developertariq/Portfolio.
+
+### Install
+#### NPM
+npm install --save-dev hint@7.x
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+Do not make any changes in config files.
+
+## Authors
+#### Md Tariq-ul Islam
+- GitHub: @developertariq
+- LinkedIn: www.linkedin.com/in/developer-tariq
+- Twitter: @developer_tariq
+
+## Contributing
+Everybody is welcome to suggest changes in the project.
+
+If you like the project, please star the project.
+
+If you spot any bugs or issues in this activity, you can open an issue with your proposed change.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Figma.com
+- @Chukwuemeka
+
+## License
+This project is MIT licensed.
```

Comment:
- [ ] [OPTIONAL] In order to make your readme professional as mentioned by a previous reviewer, I would like to recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish and add the licence link to section in your readme dedicated for it in order to keep it more professional :+1: